zulipterminal/ui_tools/boxes.py

```diff
@@ -1198,11 +1199,21 @@ def main_view(self) -> List[Any]:
 
         if any_differences:  # Construct content_header, if needed
             TextType = Dict[str, Tuple[Optional[str], str]]
-            text_keys = ('author', 'star', 'time')
+            text_keys = ('author', 'star', 'time', 'status')
             text: TextType = {key: (None, ' ') for key in text_keys}
 
             if any(different[key] for key in ('recipients', 'author', '24h')):
                 text['author'] = ('name', message['this']['author'])
+
+                email = self.message.get('sender_email', '')
+                user = self.model.user_dict.get(email, None)
+                # TODO: Currently status of bots are shown as `inactive`.
+                # Render bot users' status with bot marker as a follow-up
+                status = user.get('status', 'inactive') if user else 'inactive'
+
+                if status in STATE_ICON:
```

Comment:
Since this doesn't have an else case, line `1232` might raise an exception, right?

Reply:
To clarify, line `1203` sets the default values for all the `text_keys` (see line `1202`). So, the default value of `text['status']` would be `(None, ' ')`. Hope this helps :)

Nonetheless, I think it's good to add a note here in any case :+1:

Reply:
Thanks for clarifying.